event-stream-service/charts/event-stream-service/values.yaml

```diff
@@ -0,0 +1,204 @@
+ess:
+  nats:
+    secret:
+      name: ess-nats-auth
+  nginx:
```

Comment:
Nginx must be high availability so that cluster maintenance (pod deletion) doesn't cause a NATS outage. This will need to be configured with a minimum of three replicas.

Also, is it possible for NATS and Nginx to both be in the same anti affinity "group"? (or whatever you want to call it?) Nginx is using the old way of doing things and I don't know if the two work together. Not a deal breaker but it would be nice to not have NATS and Nginx pods on the same node.

Reply:
if you have time to prove out some configs I'm all for it. kind of swamped with the code changes so don't have enough time to spend learning to do what you're asking. both suggestions certainly sound like they would make it better.

Reply:
maybe we can tackle that in a separate PR - update these configs. we will be altering the server.conf too, to add in stream constraints (message size, persistence duration).

Reply:
A separate PR for NATS would be great!

Reply:
do you want to raise that with Danny and put it on the road map?

Reply:
I mis-read your comment, I thought you meant pull all the NATS stuff out of this PR and put it in a separate one. I'll take a look at the topology stuff and see what I can come up with for this PR.